app/page.tsx

```diff
@@ -1,27 +1,12 @@
-"use client";
+// LIBRARY IMPORTS
+import client from "@/tina/__generated__/client";
 
-//LIBRARY IMPORTS
-import { Link } from "@chakra-ui/next-js";
-import { Center, Box, Grid } from "@chakra-ui/react";
-import Image from "next/image";
+// LOCAL IMPORTS
+import Home from "./components/Home";
 
-//LOCAL IMPORTS
-import Hero from "./components/Hero";
-import CompanyValues from "./components/CompanyValues";
+export default async function Page() {
+  // CMS QUERIES
+  const result = await client.queries.page({ relativePath: "home.md" });
```

Comment:
This syntax is copied and pasted from TinaCMS docs, client is a built-in function that we import, queries is one of its properties, and page is the name we gave our collection (refer to schema in config)